### PR TITLE
Hdpi 6051 wales summ tab

### PR DESCRIPTION
**Pull request by sadmanrahh**
### Jira link

See [HDPI-6051](https://tools.hmcts.net/jira/browse/HDPI-6051)

### Change description

Make changes for Wales Summary tab:
-Add changes for the Wales Summary tab to show the Tenancy section with Wales related attributes
-Add changes to display reasons for possession draft details in Summary tab
-Add changes to have the Estate Management grounds in the same line and as comma separated, similarly to the Antisocial grounds

### Testing done

Manual testing local and preview

Screenshot 2026-05-26 at 11 25 17
Screenshot 2026-05-26 at 11 25 27
Screenshot 2026-05-26 at 11 25 35

### Security Vulnerability Assessment ###

**CVE Suppression:** Are there any CVEs present in the codebase (either newly introduced or pre-existing) that are being intentionally suppressed or ignored by this commit?
  * [ ] Yes
  * [X] No

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by toby-plunkett — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/view/builder/GroundsBuilder.java [hunk 1]
```diff
@@ -66,4 +97,16 @@ private int getSection84AConditionNumber(String label) {
...
+    private int compareEstateManagementGrounds(String firstGround, String secondGround) {
+        return Integer.compare(
+            getEstateManagementGroundNumber(firstGround),
+            getEstateManagementGroundNumber(secondGround)
+        );
+    }
+
+    private int getEstateManagementGroundNumber(String label) {
+        EstateManagementGroundsWales ground = ESTATE_MANAGEMENT_GROUND_BY_LABEL.get(label);
+        return ground == null ? 0 : ground.getRank();
```
Comment (tvr-hmcts) [inline src/main/java/uk/gov/hmcts/reform/pcs/ccd/view/builder/GroundsBuilder.java:110]: Need to add one test scenario within for this being null and returning 0

**Review by tvr-hmcts — APPROVED** [review]
LG - just one minor.

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.ccd.sdk.api.CCD;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class OccupationContractOrLicenceTabDetails {
```
Comment (gmmagruder) [inline src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java:13]: Have some crossover with this class, just put this in the shared directory and I'll align my PR with yours.

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/view/builder/GroundsBuilder.java (hunk 1 above)
Comment (sadmanrahh) [reply src/main/java/uk/gov/hmcts/reform/pcs/ccd/view/builder/GroundsBuilder.java:110]: Sure, added a test for this scenario

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java (hunk 2 above)
Comment (sadmanrahh) [reply src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java:13]: Sure, I have moved it to the shared directory now

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    @CCD(label = "What type of occupation contract or licence is in place?")
```
Comment (gmmagruder) [inline src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java]: Think this is meant to be "Occupation contract or licence agreement type"

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    private String agreementType;
+
+    @CCD(label = "What date did the occupation contract or licence begin?")
```
Comment (gmmagruder) [inline src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java]: Think this is meant to be "Occupation contract or licence start date"

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java (hunk 3 above)
Comment (sadmanrahh) [reply src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java]: That is correct, changed it now

File: src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java (hunk 4 above)
Comment (sadmanrahh) [reply src/main/java/uk/gov/hmcts/reform/pcs/ccd/domain/tabs/shared/OccupationContractOrLicenceTabDetails.java]: Yes, changed it now
